cmake/tribits/ci_support/cdash_analyze_and_report.py

```diff
@@ -327,101 +327,38 @@ def echoCmndLine(inOptions):
   echoCmndLineOptions(inOptions)
```

Comment:
Have the changes in this file been tested?

Reply:
Yes, see:

* https://github.com/TriBITSPub/TriBITS/blob/master/test/ci_support/cdash_analyze_and_report_UnitTests.py

and my comment [above](https://github.com/trilinos/Trilinos/pull/7642#discussion_r452428940).